components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -90,6 +91,8 @@ open class GleanInternalAPI internal constructor () {
         initializeCoreMetrics(applicationContext)
 
         ProcessLifecycleOwner.get().lifecycle.addObserver(gleanLifecycleObserver)
+
+        EventsStorageEngine.initialize(applicationContext)
```

Comment:
Trying this, things get a little hairy.

`EventsStorageEngine.initialize` potentially sends pings, so it needs a `PingMaker` and `PingStorageEngine` initialized.  However, `PingMaker` needs a `StorageEngineManager`, so we can't init `EventsStorageEngine` inside of `StorageEngineManager`.  Maybe changing the name of `EventsStorageEngine.initialize` to `onStartup` or something would make this clearer, that it's not really initializing -- it's doing the things it needs to do on startup once the basic glean functionality is available?

Reply:
I see - yes, let's rename to `onStartup` or `onGleanInitComplete`? We want to be specific about *when* things are expected to happen, so that we don't forget or introduce weirdness in future commits. Let's also add your comment as a code comment near the call site, please.